src/LookupModel.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $lookup)
         $this->latitude = $this->getOrNull('latitude', $lookup);
         $this->longitude = $this->getOrNull('longitude', $lookup);
         $timezone = $this->getOrNull('timezone', $lookup);
-        if (!is_null($timezone)) {
+        if (!is_null($timezone) && in_array($timezone, timezone_identifiers_list())) {
```

Comment:
If our fearless leader were to agree that changing the return value of a missing namespace would constitute a backwards incompatible change that could only be done in a new major version of the API, it would not break this client. @daspecster